config/main.py

```diff
@@ -2663,7 +2673,7 @@ def remove(ctx, interface_name, ip_addr):
             ctx.fail("'interface_name' is not valid. Valid names [Ethernet/PortChannel/Vlan/Loopback]")
         config_db.set_entry(table_name, (interface_name, ip_addr), None)
         interface_dependent = interface_ipaddr_dependent_on_interface(config_db, interface_name)
-        if len(interface_dependent) == 0 and is_interface_bind_to_vrf(config_db, interface_name) is False:
+        if len(interface_dependent) == 0 and is_interface_bind_to_vrf(config_db, interface_name) is False and get_intf_ipv6_link_local_mode(ctx, interface_name, table_name) != "enable":
```

Comment:
Why is this change required?

Reply:
This change is added to skip deletion of INTERFACE table like below one when "ipv6_use_link_local_only" is enabled.

"INTERFACE|Ethernet0": {
"ipv6_use_link_local_only": "disable",
}

If user does the steps like below :
1. config interface ipv6 enable use-link-local-only Ethernet0
2 .config interface ip add Ethernet0 1001::1/64
3. config interface ip remove Ethernet0 1001::1/64

After step 3, added new change will skip deleting the INTERFACE table as link-local is enabled in step 1.